action/action-util/src/main/java/com/powsybl/action/util/IdentifierScalable.java

```diff
@@ -0,0 +1,72 @@
+/**
+ * Copyright (c) 2017, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.action.util;
+
+import com.powsybl.commons.PowsyblException;
+import com.powsybl.iidm.network.Generator;
+import com.powsybl.iidm.network.Identifiable;
+import com.powsybl.iidm.network.Network;
+
+import java.util.List;
+import java.util.Objects;
+
+class IdentifierScalable extends AbstractScalable {
+
+
+    private final String id;
+
+    private Scalable scalable;
+
+    public IdentifierScalable(String id) {
+        Objects.requireNonNull(id);
+
+        this.id = id;
+    }
+
+    private void initScalable(Network n) {
+        Objects.requireNonNull(n);
+
+        if (scalable == null) {
+            Identifiable identifiable = n.getIdentifiable(id);
+            if (identifiable instanceof Generator) {
+                scalable = new GeneratorScalable(id);
+            } else {
+                throw new PowsyblException(identifiable.getClass() + " not support scalable yet");
```

Comment:
This message is not correct (no verb):
- identifiable.getClass() is not supported yet.
- unable to create a scalable for identifiable.getClass